src/sentry/http.py

```diff
@@ -183,3 +221,139 @@ def safe_urlopen(url, method=None, params=None, data=None, json=None,
 
 def safe_urlread(response):
     return response.content
+
+
+def expose_url(url):
+    if url is None:
+        return u'<unknown>'
+    if url[:5] == 'data:':
+        return u'<data url>'
+    url = truncatechars(url, MAX_URL_LENGTH)
+    if isinstance(url, six.binary_type):
+        url = url.decode('utf-8', 'replace')
+    return url
+
+
+def fetch_file(url, headers=None, domain_lock_enabled=True, outfile=None):
```

Comment:
I'd say if we're trying to abstract these, we should allow passing in most of the arguments for `http_session.get()` as well so that the caller can change this behavior.

Maybe change this signature to accept `**kwargs` and pass along the kwargs to `http_session.get()` with some defaults that we feel sane.

But I feel very strongly that a different caller may want to say, enable TLS verification, or set a different timeout. I think in your case for iTunes Connect, I'd say we _should_ enable TLS verification.